test/compilable/extra-files/header1i.di

```diff
@@ -14,7 +14,7 @@ out(result)
 {
 	assert(result == 0);
 }
-body
+do
```

Comment:
I'm OK with the text messages, but I'm not sure changing code (even if it's a header file).  See https://github.com/dlang/dmd/pull/7340  

I defer to wiser reviewers.